x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group1/disable.ts

```diff
@@ -151,7 +151,7 @@ export default function createDisableRuleTests({ getService }: FtrProviderContex
         savedObjects: [
           { type: 'alert', id: ruleId, rel: 'primary', type_id: 'test.cumulative-firing' },
         ],
-        message: "instance 'instance-0' has recovered due to the rule was disabled",
+        message: "instance 'instance-0' has been untracked because the rule was disabled",
```

Comment:
Would be nice to update this functional test (or add a new FT) that creates a rule with alerts, disables it and checks that the alert is marked as `untracked` and then re-enables it and sees a new alert doc created.